### PR TITLE
Add Gunicorn and event loop

### PR DESCRIPTION
**Pull request by spacetag**

### CODE REVIEW

File: code_court/courthouse/main.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+from gunicorn.six import iteritems
+
+from web import app
+from event_loop import loop
+
+def number_of_workers():
+    return (multiprocessing.cpu_count() * 2) + 1
+
+
+def handler_app(environ, start_response):
```
Comment (BenDoan) [inline code_court/courthouse/main.py]: remove this

File: code_court/courthouse/___py___event_loop.py::---.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print("https:∕∕www.youtube.com/watch?v=dQw4w9WgXcQ/")
```
Comment (BenDoan) [inline code_court/courthouse/___py___event_loop.py::---.py]: ?

Comment (spacetag) [inline code_court/courthouse/___py___event_loop.py::---.py]: I'll get rid of that!

File: code_court/courthouse/main.py (hunk 1 above)
Comment (spacetag) [inline code_court/courthouse/main.py]: Done!

**Pull request merged**